docs/instructlab-hardware-matrix.md

```diff
@@ -0,0 +1,44 @@
+# InstructLab system matrix
+
+> [!IMPORTANT]
+> The SDG and training time are always dependent on the size of your dataset.
+
+## Apple Macs
+
+|System Profile |CPU |GPU |RAM |Average SDG time |Average training time |CI support |System profile YAML file
+|---------------------------|-------|------|----------------|--------|----|-------|-----------------
+|M1 Max  |M1 |Integrated graphics |? |? |? |🟡 Supported in theory, but not regularly tested by CI |[m1_max.yaml](https://github.com/instructlab/instructlab/blob/main/src/instructlab/profiles/apple/m1/m1_max.yaml)
```

Comment:
Maybe we should have a legend for the `CI support` (I would change this to something like `Support Status`) at the top of the doc with the colors/meanings instead of replicating them in each row? So you just need to define the color circle once and can just use it in the chart

Reply:
Yeah I didn't have much time to play around with the formatting yet but will today! 
And I like that idea as well